packages/mwp-api-proxy-plugin/src/util/send.js

```diff
@@ -292,7 +292,7 @@ export function getTrackingHeaders(request) {
 	if (trackingParam) {
 		return {
 			'X-Meetup-External-Track': trackingParam,
-			'X-Meetup-External-Track-Url': `${host}${request.url.href}`,
+			'X-Meetup-External-Track-Url': `${host}${request.url.pathname}${request.url.search}`,
```

Comment:
in Hapi 17 `request.url.href` was usually, but not always, a relative path. In Hapi 18, following the WHATWG url api, `request.url.href` is always an absolute path. Therefore, changing to use `pathname` + `search` should give the equivalent of what we need.